web/spacewalk-web.changes.mackdk.issv3-fix-fqdn-label

```diff
@@ -0,0 +1,2 @@
+- Correct the organization count in the peripheral details
+- Use specific label for hub/peripheral FQDN
```

Comment:
At least for the Hub FQDN we have a bug report. So the bug number should be added

Reply:
seems there was no bug